common/persistence/dataInterfaces.go

```diff
@@ -659,10 +668,11 @@ type (
 		DeleteSignalInfo              *int64
 		UpsertSignalRequestedIDs      []string
 		DeleteSignalRequestedID       string
-		NewBufferedEvents             *SerializedHistoryEventBatch
+		NewBufferedEvents             *workflow.History
```

Comment:
I think we should not use workflow.History at this layer and directory expose collection of events as []*workflow.HistoryEvents